docs/UserManual.md

```diff
@@ -0,0 +1,28 @@
+# AAFC-DINA User Manual
+
+## Groups
+
+In DINA, groups are used to define a group of user. Mainly a natural history collection or a lab.
```

Comment:
In DINA, groups are used to define a set of users. Mainly a natural history collection or a laboratory.